### DISCUSSION TITLE
Install a Web Browser

### DISCUSSION THREAD

**Discussion by AdrielXXO** [nathaneltitane/dextop#32] [Q&A]
Hello everyone!
I would like to know if you can guide me on how to install a web browser. I know it's not recommended, but I need to use it. Please keep in mind that I don't have the slightest idea about Linux. So I will be very grateful if you can help me with patience.

I will use it to work, mainly open Excel and Google Sheets files.

**Comment by nathaneltitane** [comment]
Well, you have one of two options, direcctly under your termux session, or under the full desktop linux based container.

in either case, you should do an 'apt search <package name>' to see what is available from the repositories and if it fits with your criteria.

Once you've selected the browser, or any othe rpackage of choice, just do 'apt install <package name>'

The Browser I would recommend in either case would be the XFCE Desktop one, called Midori... then again depends what you will want to do with your browser aside for browsing?
